azure-pipelines.yml

```diff
@@ -19,7 +19,7 @@ phases:
         _testKind: Test64
 
   steps:
-    - script: build/scripts/cibuild.cmd -configuration $(_configuration) -testDesktop -$(_testKind)
+    - script: build/scripts/cibuild.cmd -configuration $(_configuration) -testDesktop -$(_testKind) -procdump
```

Comment:
Should this read `-useprocdump`?

Reply:
We probably should rename it. Given how much issues we've had getting a signed build out though I went for the minimal change here to unblock things.